docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   database:
     image: amsterdam/postgres11
     ports:
-      - 5432
+      - 5432:5432
```

Comment:
Dit kan een conflict veroorzaken op de ci build-server. Als er een test draait met een ander project dat dit heeft dan is de poort immers al bezet en zal het falen. Deze zou ik dus weer weghalen.